[BUG] can't get response when using content-type:application/json; utf-8

**Describe the bug**
I followed tutorial and found a bug
when I request server using 
curl -X GET -H "Content-Type: application/json; charset=utf-8" -d '{"name" : "sale"}' http://127.0.0.1:8181/my/service,
I got error that content-type is null
```
2020-08-05 15:15:15 [INFO ] [DssRestMasterActor.java]handlingDssRestMasterActorCommandRequest(55) : DssRestMasterActorCommandRequest: DssRestMasterActorCommandRequest{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', sender: 'rest-handler-1', methodType: 'GET', contentType: 'null', path: '/my/service', content: '{"name" : "sale"}'}
2020-08-05 15:15:15 [INFO ] [DssRestServiceActor.java]onHandlingDssRestServiceActorCommandRequest(42) : onHandlingDssRestServiceActorCommandRequest: DssRestMasterActorCommandRequest{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', sender: 'rest-handler-1', methodType: 'GET', contentType: 'null', path: '/my/service', content: '{"name" : "sale"}'}
2020-08-05 15:15:15 [WARN ] [DssRestServiceActor.java]onHandlingDssRestServiceActorCommandRequest(44) : Request content-type is null: DssRestMasterActorCommandRequest{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', sender: 'rest-handler-1', methodType: 'GET', contentType: 'null', path: '/my/service', content: '{"name" : "sale"}'}
2020-08-05 15:15:15 [ERROR] [DssRestServiceActor.java]onHandlingDssRestServiceActorCommandRequest(52) : Cannot handling request for unexpected content-type. Expected: APPLICATION_JSON, but: null
2020-08-05 15:15:15 [INFO ] [DssRestHandler.java]handlingDssRestChannelHandlerCommandResponse(212) : DssRestChannelHandlerCommandResponse: DssRestChannelHandlerCommandResponse{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', status: '400', response: 'null'}
```

--- a/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/entity/DssRestRequest.java
+++ b/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/entity/DssRestRequest.java
@@ -18,13 +18,15 @@ public class DssRestRequest implements Serializable {
     private static final long serialVersionUID = 8456589353995730809L;
     private final DssRestMethodType methodType;
     private final DssRestContentType contentType;
+    private final String charset;
     private final String uri;
     private final String content;
 
     @Builder
-    protected DssRestRequest(DssRestMethodType methodType, String uri, DssRestContentType contentType, String content) {
+    protected DssRestRequest(DssRestMethodType methodType, String uri, DssRestContentType contentType, String charset, String content) {
         this.methodType = methodType;
         this.contentType = contentType;
+        this.charset = charset;
         this.uri = uri;
         this.content = content;
     }

--- a/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java
+++ b/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java
@@ -33,12 +33,12 @@ public enum DssRestContentType {
         this.text = text;
     }
 
-    public static DssRestContentType fromText(String text) {    	
-    	try{
+    public static DssRestContentType fromText(String text) {  
+    	if(Objects.nonNull(text)){
     		return textMap.get(text.split(";")[0]);
-    	}catch(NullPointerException e) {
-    		return textMap.get(text);
     	}
+    	
+    	return null;
     }
 
     private static Map<String, DssRestContentType> initTextMap() {

--- a/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java
+++ b/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java
@@ -34,7 +34,7 @@ public enum DssRestContentType {
     }
 
     public static DssRestContentType fromText(String text) {
-        return textMap.get(text);
+    	return textMap.get(text.split(";")[0]);
     }
 
     private static Map<String, DssRestContentType> initTextMap() {

--- a/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java
+++ b/dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java
@@ -33,8 +33,12 @@ public enum DssRestContentType {
         this.text = text;
     }
 
-    public static DssRestContentType fromText(String text) {
-    	return textMap.get(text.split(";")[0]);
+    public static DssRestContentType fromText(String text) {    	
+    	try{
+    		return textMap.get(text.split(";")[0]);
+    	}catch(NullPointerException e) {
+    		return textMap.get(text);
+    	}
     }
 
     private static Map<String, DssRestContentType> initTextMap() {

